backend/akr/src/main/java/fi/oph/akr/scheduled/ExpiringAuthorisationsEmailCreator.java

```diff
@@ -49,4 +55,22 @@ public void checkExpiringAuthorisations() {
         });
     });
   }
+
+  private boolean hasEquivalentNonExpiringAuthorisation(
+    final Long expiringAuthorisationId,
+    final LocalDate expiryBetweenEnd
+  ) {
+    final Authorisation expiringAuthorisation = authorisationRepository.getReferenceById(expiringAuthorisationId);
+    final Translator translator = expiringAuthorisation.getTranslator();
+    return translator
+      .getAuthorisations()
+      .stream()
+      .filter(a -> a.getId() != expiringAuthorisationId)
+      .anyMatch(a ->
+        a.getTermEndDate().isAfter(expiryBetweenEnd) &&
+        Objects.equals(expiringAuthorisation.getFromLang(), a.getFromLang()) &&
+        Objects.equals(expiringAuthorisation.getToLang(), a.getToLang()) &&
```

Comment:
Pitäiskö korvata String-luokan equals:lla? Vai toimiiko tämä samalla tavalla?

Reply:
Objects.equals on null safe, muuten sama. Eihän ne tässä nulleja ole mutta taitaa olla parempi näin.